examples/tests/sdk_tests/bucket/metadata.test.w

```diff
@@ -1,36 +1,57 @@
 bring cloud;
+bring expect;
 
 let b = new cloud.Bucket();
 
 test "metadata()" {
-  b.put("file1.main.w", "Foo");
-  b.put("file2.txt", "Bar");
-  b.put("file3.txt", "Baz", { contentType: "application/json" });
-  b.putJson("file4.txt", "Qux");
-
-  let file1Metadata = b.metadata("file1.main.w");
-  assert(file1Metadata.size == 3);
-  assert(file1Metadata.contentType == "application/octet-stream");
+  let assertThrows = (expected: str, block: (): void) => {
+    let var error = false;
+    try {
+      block();
+    } catch actual {
+      assert(actual.contains(expected));
```

Comment:
why is it checking for "contains" and not equals?

Reply:
Same reason explained [here](https://github.com/winglang/wing/pull/4999#discussion_r1427456384).